.github/workflows/build.yaml

```diff
@@ -152,7 +152,7 @@ jobs:
         id: build
         shell: bash
         env:
-          AUTOBUILD_VCS_BRANCH: ${{ needs.setvar.outputs.branch }}
+          AUTOBUILD_VCS_BRANCH: ${{ steps.which-branch.outputs.branch }}
```

Comment:
Isn't it supposed to take channel from setvar?

Reply:
It's an old piece of code that somehow sneaked in during the merge. The current code in the develop branch uses `which-branch`.